scanner/indexer/indexer.go

```diff
@@ -312,3 +297,70 @@ func GetDigestFromReference(ref name.Reference, auth authn.Authenticator) (name.
 	}
 	return dRef, nil
 }
+
+// CreateLibIndex handles the creation of libIndex.
+func CreateLibIndex(ctx context.Context, indexerCfg config.IndexerConfig, store indexer.Store, locker *ctxlock.Locker) (*libindex.Libindex, error) {
```

Comment:
from what I can tell, this is just used for testing, so it does not need to be exposed. If you move the test to this directory, then we can lowercase this function

nit: `newLibindex`